Ctrl_AccountSearch.cls

```diff
@@ -0,0 +1,58 @@
+public with sharing class Ctrl_AccountSearch {
+	private static final String v1 = '';
```

Comment:
I think v1 is unused so we should replace it.

Reply:
done